client/components/Dropdown.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router';
+import styled from 'styled-components';
+import { remSize, prop, common } from '../theme';
+import IconButton from './mobile/IconButton';
+import Button from '../common/Button';
+
+const DropdownWrapper = styled.ul`
```

Comment:
See [Nav.jsx](https://github.com/processing/p5.js-web-editor/blob/develop/client/components/Nav.jsx), [placeholders.scss](https://github.com/processing/p5.js-web-editor/blob/develop/client/styles/abstracts/_placeholders.scss), and [_nav.scss](https://github.com/processing/p5.js-web-editor/blob/develop/client/styles/components/_nav.scss) to see how to structure the HTML/CSS/JS for a dropdown.

Reply:
This will also help with adding left and right alignment to the button that toggles open/close the dropdown!

Reply:
Passing `left` or `right` to the `<Dropdown />` component now triggers this behavior.